src/test/rgw/bucket_notification/test_bn.py

```diff
@@ -7,18 +7,20 @@
 import socket
 import time
 import os
+import io
 import string
 import boto
-from botocore.exceptions import ClientError
```

Comment:
this is causing a test failure. please put back this line

Reply:
ok, will update shortly;  thanks for the review!